packages/bpk-component-graphic-promotion/src/BpkGraphicPromo.js

```diff
@@ -53,7 +48,7 @@ export type Props = {
   onClick: () => void,
   invertVertically: boolean,
   textAlign: TEXT_ALIGN,
-  textColor: TEXT_COLORS,
```

Comment:
Have we agreed that we don't need this property anymore?

Reply:
Yes as this will be provided via `className` and the component should provide a default style OOTB 😄